Prefabs/Interactions/Interactables/SharedResources/Scripts/Grab/Action/GrabInteractableFollowAction.cs

```diff
@@ -173,6 +181,7 @@ protected virtual void OnEnable()
         /// </summary>
         protected virtual void ConfigureFollowTracking()
         {
+            IsKinematicWhenInactive = GrabSetup ? GrabSetup.Facade.ConsumerRigidbody.isKinematic : false;
```

Comment:
```suggestion
            IsKinematicWhenInactive = GrabSetup == null ? false : GrabSetup.Facade.ConsumerRigidbody.isKinematic;
```
`GrabSetup` inherits from `UnityEngine.Object`!

Too bad Unity forces us to use this null check, otherwise this could have just been `GrabSetup?.Facade.ConsumerRigidbody.isKinematic ?? false` 😞